### DISCUSSION TITLE
How to install package on windows?

### DISCUSSION THREAD

**Discussion by pintar9990** [community/community#25324] [Actions]
<p>Hello, i want to install this package on windows but i do not known how to do it, the creator said that it would work on windows and i want to try it, can someone explain me how to do it?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Installation methods and instructions can vary <em>wildly</em> depending on the package system being used. You’ll most likely need to refer to the instructions that are in the repository where you found the package you’re referring to. So, if you have a question about a project on GitHub , you should look for suggestions on how to get support in the following locations:</p>
<ul>
<li>If the project has a <code>SUPPORT</code> file, look there</li>
<li>If the project has a <code>CONTRIBUTING</code> file, look there</li>
<li>If the project has a <code>README</code> file (almost all projects on GitHub do), look there</li>
<li>If the project has none of those files or they don’t indicate where you can ask questions, open an issue on the project’s repository</li>
</ul>
<p>And if none of those bear fruit, you’ll probably want to refer to the instructions that the packaging system itself offers.</p>
<p>I hope that helps!</p>

**Reply by netmanrob** [reply]
there are no instructions. i need to know how to install better radio panel in windows 11 otherwise i will have to go back to the buggy logitech driver. i need my radio panel

**Comment by pintar9990** [comment]
<p>i mean to this project, sry i forgot send the link: <a href="https://github.com/johang/vlc-bittorrent" rel="nofollow noopener">https://github.com/johang/vlc-bittorrent</a></p>

**Comment by Json031** [comment]
Need libtorrent dependencies and C++ env:
1. Install dependencies: Pre compile the version from the libtorrent official website, or compile it yourself after installing VS.
2. Retrieve source code: ` git clone ' https://github.com/johang/vlc-bittorrent.git `.
3. Build: Generate the VS project file using CMake and build it in VS.
4. Installation: Copy the plugin files to the 'plugins' folder in VLC.
5. Test: Restart VLC, open. torrent file or magnet link.
